simple_history/manager.py

```diff
@@ -66,16 +167,37 @@ def most_recent(self):
         return self.instance.__class__(**values)
 
     def as_of(self, date):
-        """Get a snapshot as of a specific date.
+        """
+        Get a snapshot as of a specific date.
+
+        The resulting queryset will locate the most recent historical record
+        before the specified date for each primary key, generating instances.
```

Comment:
This function returns either a queryset or an instance (I'm not a fan of that, it should only do one or the other preferably, but that's how it was built originally). This variation should continue to be mentioned in the docstring somewhere. I first interpreted this such that the function would always return a queryset.

Reply:
Yes, it used to return one instance or a list of instances.  I wouldn't mind separating the concern here and actually using a breaking change to fix it up...

Reply:
I'm getting up to speed on the library. I think what you have here is fine considering it's similar to existing functionality.